app/templates/components/event-card.hbs

```diff
@@ -1,28 +1,39 @@
-<div class="ui fluid event card">
-  <a class="image" href="{{href-to 'public' event.identifier}}">
-    {{widgets/safe-image src=(if event.large event.large event.placeholderUrl)}}
-  </a>
-  <div class="content">
-    <a class="header" href="{{href-to 'public' event.identifier}}">
-      <span>{{event.name}}</span>
-    </a>
-    <div class="meta">
-      <span class="date">
-        {{moment-format event.startTime 'ddd, MMM DD HH:mm A'}}
-      </span>
+<div class="{{if isWide 'event wide ui grid row'}}">
+  {{#if isWide}}
+    <div class="ui card three wide computer six wide tablet six wide mobile column">
+      <a class="image" href="{{href-to 'public' event.identifier}}">
+        {{widgets/safe-image src=(if event.large event.large event.placeholderUrl)}}
+      </a>
     </div>
-    <div class="description">
-      {{event.shortLocationName}}
+  {{/if}}
+  <div class="ui card {{unless isWide 'event fluid' 'thirteen wide computer ten wide tablet ten wide mobile column'}}">
+    {{#unless isWide}}
+      <a class="image" href="{{href-to 'public' event.identifier}}">
+        {{widgets/safe-image src=(if event.large event.large event.placeholderUrl)}}
+      </a>
+    {{/unless}}
+    <div class="content">
```

Comment:
Reduce the extra space at the bottom:
<img width="1230" alt="screen shot 2017-05-25 at 12 52 37 am" src="https://cloud.githubusercontent.com/assets/2404372/26421692/94085b72-40e4-11e7-8d9c-119b54dbb9c1.png">

Make it like this:
<img width="1236" alt="screen shot 2017-05-25 at 12 52 26 am" src="https://cloud.githubusercontent.com/assets/2404372/26421711/a0011c3e-40e4-11e7-9eb0-1214c42da1c6.png">

Can be done by setting the height of this `content` to `130px;` (But ensure you it conditionally only for wide mode)